Dockerfile

```diff
@@ -52,12 +34,37 @@ RUN /venv/bin/pip install --upgrade pip && poetry install --no-interaction --no-
         "flower==1.0.0" \
         "tornado==6.1"
 
-RUN cd /rapidpro && npm install npm@6.14.11 && npm install
+FROM ghcr.io/praekeltfoundation/python-base-nw:3.9-bullseye
 
+ARG RAPIDPRO_VERSION
+ENV RAPIDPRO_VERSION=${RAPIDPRO_VERSION:-master}
+
+# Copy rapidpro and venv from builder
+COPY --from=builder /rapidpro /rapidpro
+COPY --from=builder /venv /venv
+ENV PATH="/venv/bin:$PATH"
+ENV VIRTUAL_ENV="/venv"
+
+# Install `psql` for `manage.py dbshell`
+# `magic` is needed since rapidpro v3.0.64
+# `pcre` is needed for uwsgi
+# `geos`, `gdal`, and `proj` are needed for `manage.py download_geojson` and `manage.py import_geojson`
 RUN apt-get-install.sh \
         postgresql-client \
         libmagic-dev \
-        libpcre3
+        libpcre3 \
+        libgeos-c1v5 \
+        libgdal28 \
+        libproj19
+
+RUN apt-get-install.sh curl && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
```

Comment:
Can we try using the version of node in the os repos? 10.x has been unsupported for nearly a year and 12.x only has a couple of months left.

Reply:
I'm going to try and see if there are any breakages with nodejs 16 (the LTS version)

Reply:
Seems like it works with nodejs 16